paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -50,6 +50,11 @@
 from paasta_tools.utils import Timeout
 from paasta_tools.utils import TimeoutError
 
+try:
```

Comment:
can you unify this with the work mmb did in deployd to make a more general metrics_provider available so that we don't have to do this for every library? i.e. take the metrics_provider config out of deployd and make it a 'global' setting?

Reply:
I've create #1621 to do that